plugins/login-resources/src/utils.ts

```diff
@@ -76,6 +76,11 @@ export async function doLogin (email: string, password: string): Promise<[Status
   }
 }
 
+function emailValidator (value: string): boolean {
```

Comment:
We also need same logic inside server/account to prevent direct API call.

Reply:
+ have noticed that this problem exists not only in SIgnUp method. In LogIn exists this mini-bug too :=)

curl log

`curl 'https://account.bold.ru/' -X POST -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:123.0) Gecko/20100101 Firefox/123.0' -H 'Accept: */*' -H 'Accept-Language: en' -H 'Accept-Encoding: gzip, deflate, br' -H 'Referer: https://app.bold.ru/' -H 'Content-Type: application/json' -H 'Origin: https://app.bold.ru' -H 'Connection: keep-alive' -H 'Sec-Fetch-Dest: empty' -H 'Sec-Fetch-Mode: cors' -H 'Sec-Fetch-Site: same-site' -H 'Sec-GPC: 1' -H 'TE: trailers' --data-raw '{"method":"login","params":["123","123"]}'`

Reply:
@haiodo I have added it for the server side as the same for UI, but as I haven't deep knowledge of your product could you please check?)